assets/scripts/passwd-strength-indicator.js

```diff
@@ -0,0 +1,63 @@
+(function (window, document) {
+  const indicator = document.getElementById('password-strength')
+
+  document.getElementById('password').addEventListener('input', function(event) {
+    const strength = getStrength(event.target.value)
+    indicator.value = parseInt(strength.percentage, 10)
+    indicator.setAttribute('class', `pw-${strength.label}`)
+  }, false)
+
+  // Return given password srength as an object {percentage, label}
+  getStrength = function (password) {
+    if (!password && password !== '') {
```

Comment:
For me, the surprising thing is to test `!password`, not the `password !== ''` exception